spec/node/asciidoctor.spec.js

```diff
@@ -9,11 +9,20 @@ var config = {
   }
 };
 var asciidoctor = require('../../build/asciidoctor.js')(config);
+function asciidoctorVersionGreaterThan (version) {
```

Comment:
Adding an API to get the version would be consistent with AsciidoctorJ.

http://www.javadoc.io/doc/org.asciidoctor/asciidoctorj/1.6.0-alpha.3

Though I'd argue that should be `getAsciidoctorVersion`.

As far as comparison goes, it follows the Gem::Version rules. See http://ruby-doc.org/stdlib-2.4.1/libdoc/rubygems/rdoc/Gem/Version.html#method-i-3C-3D-3E. That comparison will be useful to have somewhere in Opal or Asciidoctor.js anyway.